feat: display application runtime

feloy opened the pull request:
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Display application runtime

The models manager needs to be initialized synchronously, so the app manager has an already initialized models catalog (or depending on race condition, it is possible that the model's backend is not found)

### Screenshot / video of UI
![display-app-runtime](https://github.com/user-attachments/assets/b749c8bf-35b8-4c46-b279-8c74a5a29a25)

### What issues does this PR fix or reference?

Part of #2614 

### How to test this PR?

Start a recipe and check that the runtime is correctly displayed
Restart (PD, extension, container runtime, etc) and check that the runtime is still correct

gastoner reviewed (approved):
LGTM on Linux

jeffmaury reviewed (commented):
For me this PR addresses 2 concerns:
- models manager loading
- application runtime

The first part is not complete as it is caused by the caatlog tio be loaded asynchroniusly (see #2937)

Can we split it into 2 PRs ?

feloy commented:
> For me this PR addresses 2 concerns:
> 
> * models manager loading
> * application runtime
> 
> The first part is not complete as it is caused by the caatlog tio be loaded asynchroniusly (see #2937)
> 
> Can we split it into 2 PRs ?

Right, I removed the "models manager loading" part, I'll create a new PR with this part

feloy commented:
@jeffmaury is it ok as this?

jeffmaury reviewed (approved):
LGTM
